data/card_catalogue.json

```diff
@@ -403,13 +403,13 @@
           }
         },
         {
-          "title": "Monk Mode",
+          "title": "Bashbolster",
```

Comment:
What was Monk Mode referencing? DnD?

Reply:
That and some fringe internet groups we may not want to end up associated with